src/Modules/OrchardCore.Commerce/Drivers/PriceVariantsPartDisplayDriver.cs

```diff
@@ -16,13 +18,16 @@ public class PriceVariantsPartDisplayDriver : ContentPartDisplayDriver<PriceVari
 {
     private readonly IMoneyService _moneyService;
     private readonly IPredefinedValuesProductAttributeService _predefinedValuesProductAttributeService;
+    private readonly IOptions<CommerceSettings> _options;
```

Comment:
Please give it a more specific name so we don't need to scroll up here to figure out what kind of options it is.